client/gutenberg/editor/index.js

```diff
@@ -8,11 +8,13 @@ import {
 	siteEditor,
 	exitPost,
 	redirectSiteEditor,
+	redirectToPermalinkIfLoggedOut,
 } from './controller';
 
 export default function () {
 	page(
 		'/site-editor/:site?',
+		redirectLoggedOut,
```

Comment:
cc @mattwiebe: does this need to be further tested with regards to the recent site editor un-framing in p7DVsv-h8i-p2 ?

Reply:
@tyxla @enejb I don't think that what this PR is doing should have anything to do with the Site Editor - this is definitely out of date with the [new routing](https://github.com/Automattic/wp-calypso/blob/fd388ceede5996f6480bf6cc41eaf4e8b313d0d1/client/gutenberg/editor/index.js#L7) which is just

```js
page( '/site-editor/:site?', siteSelection, redirectSiteEditor );
```

And with https://github.com/Automattic/jetpack/pull/30072 due to land next week, the `/site-editor/SITE` route will cease most usage anyway, it'll just stay there for back compat and for whatever places hardcode the `/site-editor` route in Calypso.

Reply:
Thank you for confirming @mattwiebe 🙌